quill-core/src/main/scala/io/getquill/Query.scala

```diff
@@ -47,6 +47,7 @@ sealed trait EntityQuery[T]
   extends Query[T] {
 
   def apply(alias: String, propertyAlias: (T => (Any, String))*): EntityQuery[T]
+  def schema(f: Schema[T] => Any): EntityQuery[T]
```

Comment:
@lvicentesanchez yes, I started with schema keyword so previous tests would continue to work.